### PR TITLE
chore(stageTemplate): fix template form error with field arguments & cmd

### PR DESCRIPTION
**Pull request by huxiangtao**
**What this PR does / why we need it**:

fix template form error with field arguments & cmd , exchange field spec.pod.inputs.arguments with spec.pod.container fileds

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @cd1989 

**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by supereagle** [comment]
> /hold

Please give the reason when hold others' PRs

File: web/src/components/stageTemplate/add/Form.js [hunk 1]
```diff
@@ -55,13 +55,24 @@ export default class StageTemplateForm extends React.Component {
     };
   };
 
+  formatCmdValue = val => {
+    return val.replace(/;/g, '\n');
```
Comment (cd1989) [inline web/src/components/stageTemplate/add/Form.js]: `val.replace(/;\s+/g, ';\n')`

**Pull request merged**